cadc-util/src/intTest/java/ca/nrc/cadc/db/version/KeyValueDAOTest.java

```diff
@@ -0,0 +1,210 @@
+ *  the GNU Affero General Public        la “GNU Affero General Public
+ *  License as published by the          License” telle que publiée
+ *  Free Software Foundation,            par la Free Software Foundation
+ *  either version 3 of the              : soit la version 3 de cette
+ *  License, or (at your option)         licence, soit (à votre gré)
+ *  any later version.                   toute version ultérieure.
+ *
+ *  OpenCADC is distributed in the       OpenCADC est distribué
+ *  hope that it will be useful,         dans l’espoir qu’il vous
+ *  but WITHOUT ANY WARRANTY;            sera utile, mais SANS AUCUNE
+ *  without even the implied             GARANTIE : sans même la garantie
+ *  warranty of MERCHANTABILITY          implicite de COMMERCIALISABILITÉ
+ *  or FITNESS FOR A PARTICULAR          ni d’ADÉQUATION À UN OBJECTIF
+ *  PURPOSE.  See the GNU Affero         PARTICULIER. Consultez la Licence
+ *  General Public License for           Générale Publique GNU Affero
+ *  more details.                        pour plus de détails.
+ *
+ *  You should have received             Vous devriez avoir reçu une
+ *  a copy of the GNU Affero             copie de la Licence Générale
+ *  General Public License along         Publique GNU Affero avec
+ *  with OpenCADC.  If not, see          OpenCADC ; si ce n’est
+ *  <http://www.gnu.org/licenses/>.      pas le cas, consultez :
+ *                                       <http://www.gnu.org/licenses/>.
+ *
+ *  : 5 $
+ *
+ ************************************************************************
+ */
+
+package ca.nrc.cadc.db.version;
+
+import ca.nrc.cadc.db.ConnectionConfig;
+import ca.nrc.cadc.db.DBConfig;
+import ca.nrc.cadc.db.DBUtil;
+import ca.nrc.cadc.db.DatabaseTransactionManager;
+import ca.nrc.cadc.util.Log4jInit;
+import java.sql.SQLException;
+import java.util.List;
+import javax.sql.DataSource;
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+public class KeyValueDAOTest {
+    private static final Logger log = Logger.getLogger(KeyValueDAOTest.class);
+
+    private static final String SCHEMA = "dbversion";
+    private static final String DATABASE = "cadctest";
+    private static final String MODELNAME = "KeyValueDAOTest";
+    private static final String MODELVERSION = "0.1";
+    private static final String TABLE_NAME = SCHEMA + ".KeyValue";
+    private static DataSource dataSource;
+
+    private static final String CREATE = "create table dbversion.KeyValue " +
+            "(name varchar(32) not null primary key,value varchar(32) not null,lastModified timestamp not null)";
+
+    public KeyValueDAOTest() {
+    }
+
+    @BeforeClass
+    public static void setup() {
+        Log4jInit.setLevel("ca.nrc.cadc.db.version", Level.DEBUG);
+
+        try {
```

Comment:
setup( ) methods don't need to `assert` anything, just throw exceptions if they fail

Reply:
Updated to throw an exceptions encountered during setup.